components/ProfilePage/ProfilePage.js

```diff
@@ -84,6 +94,16 @@ function ProfilePage({ id, slug }) {
     }
   }, [latestSlug, router]);
 
+  // Load current-user-only fields when currentUser changes
+  useEffect(() => {
```

Comment:
I think we can add [`refetchQueries: ['LoadProfilePage']`](https://www.apollographql.com/docs/react/data/mutations/#options) to `useMutation(UPDATE_USER)`  call in `EditAvatarDialog` instead. In this way, profile page data can be reloaded after user submits avatar.

Reply:
I updated user cached instead so it's updated in all avatars